gamelogs-client/src/App.tsx

```diff
@@ -1,35 +1,171 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+interface NewsItem {
+  title: string;
+  date: string;
+  thumbnail: string;
+}
+interface Game {
+  title: string;
+  logo: string;
+  thumbnail: string;
+}
+interface GameCategory {
+  title: string;
+  games: Game[];
+}
+
+const news: NewsItem[] = [
+  { title: 'News One', date: 'xx/xx/xxxx', thumbnail: '#8D6ED7'},
+  { title: 'News Two', date: 'xx/xx/xxxx', thumbnail: '#D76E6E' },
+  { title: 'News Three', date: 'xx/xx/xxxx', thumbnail: '#6ED771' },
+  { title: 'News Four', date: 'xx/xx/xxxx', thumbnail: '#D3D76E' },
+  { title: 'News Five', date: 'xx/xx/xxxx', thumbnail: '#6EC7D7' },
+  { title: 'News Six', date: 'xx/xx/xxxx', thumbnail: '#8D6ED7' },
+];
+
+const games: Game[] = [
+  { title: 'Game One', logo: 'logo', thumbnail: '#8D6ED7' },
+  { title: 'Game Two', logo: 'logo', thumbnail: '#D3D76E' },
+  { title: 'Game Three', logo: 'logo', thumbnail: '#6EC7D7' },
+  { title: 'Game Four', logo: 'logo', thumbnail: '#6ED771' },
+  { title: 'Game Five', logo: 'logo', thumbnail: '#D76E6E' },
+  { title: 'Game Six', logo: 'logo', thumbnail: '#8D6ED7' },
+  { title: 'Game Seven', logo: 'logo', thumbnail: '#D3D76E' },
+  { title: 'Game Eight', logo: 'logo', thumbnail: '#6EC7D7' },
+  { title: 'Game Nine', logo: 'logo', thumbnail: '#6ED771' },
+  { title: 'Game Ten', logo: 'logo', thumbnail: '#D76E6E' },
+];
+
+const categories: GameCategory[] = [
+  {
+    title: 'Top Rated',
+    games: games,
+  },
+  {
+    title: 'Your List',
+    games: games,
+  },
+  {
+    title: 'Relevant Genre 1',
+    games: games,
+  },
+  {
+    title: 'Relevant Genre 2',
+    games: games,
+  },
+  {
+    title: 'Relevant Genre 3',
+    games: games,
+  },
+];
 
 function App() {
-  const [count, setCount] = useState(0)
+  return (  
+    <div className="w-screen h-screen">
+      <main className="h-full">
+        
+        {/*Hero Section*/}
```

Comment:
Could make these sections into separate components. Would help with testing and just logic separation / code readability (don't need comments when it's a separate function).

Reply:
I'll merge this commit, but I may make the sections into components in the future.